src/openzaak/notifications/models.py

```diff
@@ -18,6 +18,8 @@ class FailedNotification(StatusLog):
     status_code = models.IntegerField(
         _("status_code"),
         help_text=_("Status code received from the Notifications API."),
+        null=True,
+        blank=True,
```

Comment:
We don't log HTTP status anymore (I don't think we did log it before tbh), and admin resend is deprecated, so I just marked `status` as optional before removing this whole model in the next OZ version